oraclelinux/node12/Dockerfile

```
@@ -1,15 +1,15 @@
 FROM        oraclelinux:7-slim
 MAINTAINER  Apiary <sre@apiary.io>
 
-ENV REFRESHED_AT 2020-11-02
+ENV REFRESHED_AT 2020-11-13
 ENV NODEJS_VERSION_MAJOR 12
 ENV NPM_VERSION 6.13.4
 
 USER root
 
 RUN yum install -y oracle-nodejs-release-el7 \
     && yum-config-manager --enable ol7_developer_nodejs${NODEJS_VERSION_MAJOR} \
-    && yum install -y nodejs \
+    && yum install -y nodejs-12.* \
```

```suggestion
    && yum install -y nodejs-${NODEJS_VERSION_MAJOR}.* \
```

We can reference the variable again, makes it a bit easier to copy, and make future node images as only one place to update version. Only the three lines 3/5/6 need to differ per dockerfile for node versions.

---

Thanks. Just did that for both images.